cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -471,85 +470,132 @@ describe('Data Table', () => {
+        hasClassAssertion('@sortIcon', `${ARROW_SORT_CLASS}`);
         cy.get('@firstCell')
           .click()
           .then(() => {
-            hasClassAssertion('@sortIcon', 'icon-arrow-up');
-            cy.get('@firstRow').should('contain', 'Name 1');
+            hasClassAssertion('@sortIcon', `${ARROW_UP_CLASS}`);
+            cy.get('@rows')
+              .eq(0)
+              .should('contain', 'Name 1');
             cy.get('@sortIcon').should('have.css', 'transform', 'none');
           });
         cy.get('@firstCell')
           .click()
           .then(() => {
-            hasClassAssertion('@sortIcon', 'icon-arrow-up');
+            hasClassAssertion('@sortIcon', `${ARROW_UP_CLASS}`);
             cy.get('@sortIcon')
               .should('have.attr', 'style')
               .should('contain', 'transform: rotate(180deg);');
-            cy.get('@firstRow').should('contain', 'Surname 2');
+            cy.get('@rows')
+              .eq(0)
+              .should('contain', 'Surname 2');
           });
         cy.get('@firstCell')
           .click()
           .then(() => {
-            cy.get('@firstRow').should('contain', 'Name 1');
-            hasClassAssertion('@sortIcon', 'icon-arrow-sort');
+            cy.get('@rows')
+              .eq(0)
+              .should('contain', 'Name 1');
+            hasClassAssertion('@sortIcon', `${ARROW_SORT_CLASS}`);
           });
       });
 
       it(`should sort by status using a custom template`, () => {
-        cy.get('@firstRow').should('contain', 'Name 1');
+        const indexes = [0, 1, 2];
+        indexes.forEach(index => {
+          if (index % 2 === 0) {
+            cy.get('@rows')
+              .eq(index)
+              .should('contain', 'active');
+          } else {
+            cy.get('@rows')
+              .eq(index)
+              .should('contain', 'inactive');
+          }
+        });
+
         cy.get(`[data-cy='data-table-sorting'] .${DATA_TABLE_CLASSES.ROW}`)
           .eq(0)
           .find(`.${DATA_TABLE_CLASSES.CELL}`)
           .eq(1)
           .as('idCell')
           .click()
           .then(() => {
-            cy.get('@firstRow').should('contain', 'active');
+            const actives = [0, 1, 2];
+            actives.forEach(index => {
+              cy.get('@rows')
+                .eq(index)
+                .should('contain', 'active');
+            });
           });
         cy.get('@idCell')
           .click()
           .then(() => {
-            cy.get('@firstRow').should('contain', 'inactive');
+            const inactives = [0, 1, 2];
+            inactives.forEach(index => {
+              cy.get('@rows')
+                .eq(index)
+                .should('contain', 'inactive');
+            });
           });
         cy.get('@idCell')
           .click()
           .then(() => {
-            cy.get('@firstRow').should('contain', 'active');
+            const indexes = [0, 1, 2];
+            indexes.forEach(index => {
```

Comment:
Is this to check no-sorting scenario ? In that case, could you define a function and reuse it in both cases?